src/client/src/ui/spotTile/views/spotTileView.jsx

```diff
@@ -66,6 +87,12 @@ export default class SpotTileView extends ViewBase {
     );
   }
 
+  _displayCurrencyChart(){
+    router.publishEvent(this.props.modelId, 'displayCurrencyChart', {});
+    let timeoutId = setTimeout(()=> this.setState({currencyChartIsOpening: false}), 2000);
```

Comment:
just thinking you might want to run the hack code first then publish the event to launch open fin.